Description: Avoid implicit JSON type conversion
Author: Oliver Reiche <oliver.reiche@gmail.com>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: b/src/other_tools/just_mr/setup_utils.cpp
===================================================================
--- a/src/other_tools/just_mr/setup_utils.cpp
+++ b/src/other_tools/just_mr/setup_utils.cpp
@@ -205,7 +205,7 @@ auto ReadConfiguration(
                                 absent_file_opt->string());
                     std::exit(kExitConfigError);
                 }
-                absent_set.insert(repo);
+                absent_set.insert(repo.get<std::string>());
             }
             auto new_repos = nlohmann::json::object();
             auto repos = config.value("repositories", nlohmann::json::object());
